### ISSUE TITLE
Robustness Enhanced model

### ISSUE THREAD

**Issue by Mumuwei** [Georgefwt/AquaLoRA#17]
Thank you for your wonderful work. 
May I ask what the differences are between the Prior-preserving Fine-tuned and Robustness Enhanced models?

**Comment by Georgefwt** [comment]
That's a great question.

The motivation behind this comes from an observation. We trained the model on $512 \times 512$ images, and it performs well at that sampling resolution. However, when we increased the sampling resolution (e.g., to $768 \times 640$) and simply rescaled the image back to $512 \times 512$ for the decoder, the extraction failed. Interestingly, we discovered that if we cropped $256 \times 256$ patches from the four corners and stitched them together, the extraction accuracy approached that of the native $512 \times 512$ resolution (see pic below). We suspect this behavior is caused by the upsample/downsample convolution layers in the Stable Diffusion architecture (which might not be same in DiT-based models).

<img width="775" height="299" alt="Image" src="https://github.com/user-attachments/assets/f69d9d14-cee1-4096-9a6f-41281806eb79" />

Since we can't expect users to perform such specific cropping operations during extraction, we introduced this "four-corner cropping" distortion during the Latent Watermark Pre-training stage. This forces the watermark decoder to learn to pay attention to the corners, improving robustness across different sampling resolutions. We call this process Robustness Enhancement. For more details, you can refer to Appendix D of our paper.

**Comment by Mumuwei** [comment]
Thank you very much for your answer. Further, I have a question. If the watermark information is focused on the four corners, then when extracting the watermark, if crop is used to remove the four corners, can the watermark be extracted correctly?

**Comment by Georgefwt** [comment]
That's a valid concern. To be honest, this is currently an unresolved issue. Losing corner via cropping does affect extraction.

But, keep in mind that SD1.5 struggles with quality when scaling up resolution, which adds another layer of difficulty. It would be interesting to see if DiT-based models avoid this problem since they don't have the same U-Net convolution biases, but we haven't tested that yet.

I would love to see follow up research that tackle this problem!

**Comment by Mumuwei** [comment]
I see. Thanks a lot.
